### PR TITLE
Added test serialize change threshold multisig

### PR DESCRIPTION
**Pull request by rllola**
Added a test to make sure the change threshold params can be properly serialized.

### CODE REVIEW

File: examples/wasm_node/test/test_multisig.js [hunk 1]
```diff
@@ -232,3 +232,15 @@ describeCall('cancelMultisig', function() {
     assert(JSON.parse(signature).Signature)
   })
 })
+
+describeCall('serializeParams reported cases', function() {
+  it('newThreshold', function () {
+    const innerParams = {
+      NewTreshold: 2
```
Comment (jleni) [inline examples/wasm_node/test/test_multisig.js:239]: There is a typo here
```suggestion
      NewThreshold: 2
```

**Review by jleni — CHANGES_REQUESTED** [review]

File: examples/wasm_node/test/test_multisig.js (hunk 1 above)
Comment (jleni) [reply examples/wasm_node/test/test_multisig.js:239]: I wonder how/why the test passes with that typo there...

Comment (rllola) [reply examples/wasm_node/test/test_multisig.js:239]: yeah that's really weird I didn't notice

Comment (rllola) [reply examples/wasm_node/test/test_multisig.js:239]: Apparently I am really consistent in my typos :
https://github.com/Zondax/filecoin-signing-tools/blob/master/signer/src/api.rs#L366

**Pull request merged**